src/main/java/core/basesyntax/service/RegistrationException.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service;
+
+public class RegistrationException extends RuntimeException {
+    public RegistrationException(String message) {
+        super("User not registered. " + message + " Please, check input data and try again.");

```

Comment:
It is bad practice to change the message in the exception, it is better to output what is passed in the message